ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/productdetails/productdetails.html

```diff
@@ -0,0 +1,31 @@
+<div class ='product-details-cmp'>
```

Comment:
@Rajamouli-hcl Spacing issue between price and product details.

Reply:
fixed spacing issue as per your comments